apps/privacy-panel/js/ala/define_custom_location.js

```diff
@@ -142,15 +142,23 @@ function(panels, SettingsListener, SettingsHelper) {
       // set new list of cities for selected region
       this.selectedRegionCities = this.regionsAndCities[this.config.region];
 
+      // sort regions by their localized name
+      var keys = Object.keys(this.regionsAndCities);
+
+      keys.sort(function(a, b) {
+        return (navigator.mozL10n.get(a) > navigator.mozL10n.get(b));
+      });
```

Comment:
you still try to compare using ">" between two strings. That's not how you compare two strings in Unicode.